wfe/web-front-end.go

```diff
@@ -157,10 +147,6 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h wfe
 
 			switch request.Method {
 			case "HEAD":
```

Comment:
Could you add a comment here that Go's `net/http` server will strip out responses to HEAD requests?